cmd/cluster-init/update_jobs.go

```diff
@@ -21,24 +25,75 @@ const (
 	generator    jobconfig.Generator = "cluster-init"
 )
 
+type BuildClusters struct {
+	Managed []string `json:"managed,omitempty"`
+}
+
 func updateJobs(o options) error {
+	var clusters []string
+	if o.clusterName == "" {
+		// Updating ALL cluster-init managed clusters
+		buildClusters, err := loadBuildClusters(o)
```

Comment:
Could we derive this from the bootstrap's config? like https://github.com/openshift/ci-tools/pull/2378/files#diff-5b92b4f3254b078d3a1a836f7055b073b60ddc08f024494639503b0052188d8dR190

Reply:
I thought about that as well, but I thought that a first-class config file would be less hacky here.